spec/dor/identifiable_spec.rb

```diff
@@ -312,4 +312,14 @@ class IdentifiableItem < ActiveFedora::Base
       expect(item.get_related_obj_display_title(mock_apo_obj, mock_default_title)).to eq(mock_default_title)
     end
   end
+
+  describe '#adapt_to_cmodel' do
+    context 'for a Hydrus collection' do
```

Comment:
👌 . I don't think we need to be too exhaustive, although maybe I'm too optimistic about the coverage on `Dor.find`.